share/local_dev_values.yaml

```diff
@@ -0,0 +1,408 @@
+### Common Configuration ###
```

Comment:
What's your thinking with this file? I'm not sure what it's trying to do here. If there are specific settings that need to be changed for local dev, we should either build them into the templates or limit this file to only the overrides.

Reply:
I guess maybe I was stuck in a mode of thinking that we provide examples to our customers ... I think it's best if we left this out.